anyway/infographics_utils.py

```diff
@@ -1118,91 +1121,67 @@ def localize_items(request_params: RequestParams, items: Dict) -> Dict:
-                "severe_injury_severity_count": 3,
-                "killed_injury_severity_text": "הרוג",
-                "killed_injury_severity_count": 0,
-            },
-            {
-                "year": 2010,
-                "light_injury_severity_text": "פצוע קל",
-                "light_injury_severity_count": 24,
-                "severe_injury_severity_text": "פצוע קשה",
-                "severe_injury_severity_count": 0,
-                "killed_injury_severity_text": "הרוג",
-                "killed_injury_severity_count": 1,
-            },
-            {
-                "year": 2011,
-                "light_injury_severity_text": "פצוע קל",
-                "light_injury_severity_count": 9,
-                "severe_injury_severity_text": "פצוע קשה",
-                "severe_injury_severity_count": 2,
-                "killed_injury_severity_text": "הרוג",
-                "killed_injury_severity_count": 1,
-            },
-            {
-                "year": 2012,
-                "light_injury_severity_text": "פצוע קל",
-                "light_injury_severity_count": 21,
-                "severe_injury_severity_text": "פצוע קשה",
-                "severe_injury_severity_count": 2,
-                "killed_injury_severity_text": "הרוג",
-                "killed_injury_severity_count": 4,
-            },
-            {
-                "year": 2013,
-                "light_injury_severity_text": "פצוע קל",
-                "light_injury_severity_count": 21,
-                "severe_injury_severity_text": "פצוע קשה",
-                "severe_injury_severity_count": 2,
-                "killed_injury_severity_text": "הרוג",
-                "killed_injury_severity_count": 4,
-            },
-            {
-                "year": 2014,
-                "light_injury_severity_text": "פצוע קל",
-                "light_injury_severity_count": 10,
-                "severe_injury_severity_text": "פצוע קשה",
-                "severe_injury_severity_count": 0,
-                "killed_injury_severity_text": "הרוג",
-                "killed_injury_severity_count": 1,
-            },
-            {
-                "year": 2015,
-                "light_injury_severity_text": "פצוע קל",
-                "light_injury_severity_count": 13,
-                "severe_injury_severity_text": "פצוע קשה",
-                "severe_injury_severity_count": 2,
-                "killed_injury_severity_text": "הרוג",
-                "killed_injury_severity_count": 0,
-            },
-        ]
+            self.items = add_empty_keys_to_gen_two_level_dict(
+                self.convert_to_dict(query.all()),
+                InjurySeverity.codes(),
+                list(range(self.request_params.start_time.year,
+                           self.request_params.end_time.year + 1))
+            )
 
+        except Exception as e:
+            logging.error(f"InjuredAccidentsWithPedestriansWidget.generate_items(): {e}")
+            raise Exception(e)
```

Comment:
See a way to preserve more information from the source exception - [see the marked ansewr](https://stackoverflow.com/questions/696047/re-raise-exception-with-a-different-type-and-message-preserving-existing-inform#792163)

Reply:
the issue in SO relates to re-raising an exception of a different type, here I am not doing that, i'm re-raising the same type. actually, the only reason i'm doing this is because nowhere down the line is there any code which prints out the exception details